### DISCUSSION TITLE
Next JS not running on bun

### DISCUSSION THREAD

**Discussion by anandhunt** [oven-sh/bun#5539] [Q&A]
i ran next js project on bun , using command ,  bun --bun run dev, but i got this error [bun] Warning: worker_threads.Worker option "eval" is not implemented.
[bun] Warning: worker_threads.Worker option "stdout" is not implemented.
[bun] Warning: worker_threads.Worker option "stderr" is not implemented.
[bun] Warning: worker_threads.Worker option "resourceLimits" is not implemented.

**Comment by Adamkaram** [comment]
bunx create-next-app ProjectName
will work just as fine

**Reply by ElusiveMind** [reply]
Not sure this answered the question. They're not talking about starting a new app, but running an existing one. If I am reading correctly.

**Comment by jimmywarting** [comment]
What really needs to be addressed is for https://github.com/nodejs/node/issues/43583 to be addressed, and then for NextJS to start using speced whatwg worker instead. give that issue a 👍
